types.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2014 The Go Authors.  All rights reserved.
```

Comment:
Is this the correct copyright header?

Reply:
Yes, the work was started by Dmitri Shuralyov and was abandoned on his github page. I kept his commit and continued on top.

Reply:
But this file is added in a commit that you authored, unless I misread the history https://github.com/fyne-io/gl-js/commit/b72f4b5be1d982188a0d27fd8bc09be5ea7339c7#diff-b990161986c40e0867e71c60779bc98a8730a9eb0ca5866ae8189e126863c059

Reply:
Hum, I see. Not sure in this case what should be the copyright of this file. It is the result of refactoring code across multiple file that come from Dmitri initial commit. Any idea?

Reply:
Can't it just be omitted? Many files in this repo have no copyright header.
The assertion of copyright explicitly is not really required in many jurisdictions and Fyne has never included it per-file. Copyright automatically sits with the author(s) unless it has explicitly been assigned to someone else.

Reply:
Will do that. As long as Git keep history, it should be correct.